File: A2.py
Lista = []
sw = True
items = []


while sw :

	Texto = input()
	
	if Texto != "0":
		Lista.append(Texto.split())
	else:
		sw = False


for x in Lista:
	resp = ""
	
	for j in range(int(x[1]), int(x[2]) + 1):
		if j % int(x[0]) == 0:
			resp = resp + str(j) + " "


	print(resp)

File: Múltiplos de un número dentro de un rango/A2.js
const input = require('readline-sync')

let Lista = []
let sw = true
items = []

while (sw) {
    const Texto = input.question('')
    if(Texto != '0') {
        Lista.push(Texto.split(' '))
    } else {
        sw = false
    }
}

Lista.forEach((row) => {
    let resp = ''
    for(let j = parseInt(row[1]); j <= parseInt(row[2]); j++) {
        if (j % parseInt(row[0]) == 0){
            resp = resp + j.toString() + ' '
        }
    }
    console.log(resp)
})